app/components/hero_section_component.rb

```diff
@@ -3,7 +3,8 @@
 class HeroSectionComponent < ViewComponent::Base
   def initialize(image_path:, image_credit:, title:, alttext:)
     super
-    @title = title.presence || 'PlaceCal is a community events calendar where you can find everything near you, all in one place.'
+    # TODO: remove hard coded string from component
+    @title = title.presence || "PlaceCal is a community events calendar where you can find out everything that's happening, all in one place."
```

Comment:
@katjam - when i try and replace this with a locale string, it gives me an error that that function isn't availabe here and some instructions if that's what I really want to do. is this to prevent an antipattern do you think, like should I be approching this in a different way, or should i just get on with it and include the translation library here so i can use the string? does that make sense?

Reply:
I think you should include the translation library.

Headers are a weird case I guess because patternwise, you are right, the translations should be evaluated in the html templates.

Reply:
right, cool. but then it would be very anti-DRY to have the translation string fed through every time its called by the template. glad you agree its an edge case i couldnt quite make sense of the error message